.env.dusk.local

```diff
@@ -1,14 +1,14 @@
-APP_NAME="Meteostanice LKBASK - Hodoňovice"
```

Comment:
as I said in #12, put this file into gitignore

Reply:
added to .gitgnore